en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md

```diff
@@ -65,6 +65,19 @@ Let's start the servers in the given order.
 !!! Warning
     Be sure to [configure the SSL certificates](../../setup/observability/setting-up-classic-observability-deployment/#configure-ssl-certificates) before starting the Micro Integrator.
 
+#### Configuring the Micro Integrator to Publish Data to Analytics Server
```

Comment:
@ramindu90 - shall we add this section to the 'Setting up classic observability deployment' doc?
according to how our doc is structured, we have all the 'setting up content in one place..